app/containers/App/index.js

```diff
@@ -118,7 +113,7 @@ App.defaultProps = {
 }
 
 const mapStateToProps = createStructuredSelector({
```

Comment:
Missed an opportunity to remove the selects, I've been trying to get rid for a while now.